snow/engine/snowman/transitive.go

```diff
@@ -460,7 +461,7 @@ func (t *Transitive) issueFrom(vdr ids.ShortID, blk snowman.Block) (bool, error)
 	// issue [blk] and its ancestors to consensus.
 	// If the block has been issued, we don't need to issue it.
 	// If the block is queued to be issued, we don't need to issue it.
```

Comment:
// If the block is at a height lower than the last accepted block, we don't need to issue it.